### PR TITLE
chore: update package-lock.json

### PR DESCRIPTION
**Pull request by nolanmar511**
Also ran "npm run fix". The version of gts we're using changed with this change.

### CODE REVIEW

File: ts/src/profile-serializer.ts [hunk 1]
```diff
@@ -156,9 +156,7 @@ function serialize<T extends ProfileNode>(
         profLoc = sourceMapper.mappingInfo(profLoc);
       }
     }
-    const keyStr = `${node.scriptId}:${profLoc.line}:${profLoc.column}:${
-      profLoc.name
-    }`;
+    const keyStr = `${node.scriptId}:${profLoc.line}:${profLoc.column}:${profLoc.name}`;
```
Comment (aalexand) [inline ts/src/profile-serializer.ts]: This seems an unrelated change, is this expected?

Comment (nolanmar511) [reply ts/src/profile-serializer.ts]: The version of gts used seem to change with this update; I had to run `npm run fix` so unit tests passed.

Noted this in the description.
